frontend/src/seqvars/queries/seqvarQueryPresetsFrequency.ts

```
@@ -0,0 +1,210 @@
+      if (!!context?.previousValue) {
+        queryClient.setQueryData(
+          seqvarsApiQuerypresetsfrequencyRetrieveOptions({
+            path: variables.path,
+          }).queryKey,
+          context.previousValue,
+        )
+      }
+    },
+    // Always refetch after error or success.
+    onSettled: (data) => {
+      if (!!data) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)